ingestion/combine_data.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd # pip install pandas
+import os
+
+# list constants that hold the identified similiar names
+LIST_ISSUE_DATE = {"issue_date", "job_start_date", "issued_date", "permitissuedate"}
+LIST_CONTRACTOR = {"contractor_company_name", "contact_1_name","firm_name", "contractor_name", "contractorname"}
+LIST_LATITUDE =  {"latitude","gis_latitude", "lat"}
+LIST_LONGITUDE = {"longitude","gis_longitude", "lng"}
+
+# dataframe to store the combined data of all the cities after standardization
+combined_df = pd.DataFrame()
+
+# pointing towards the directory with our stripped data
+os.chdir("./stripped_data")
+
+# for loop that will append the data to the lists
+for file in os.listdir(os.getcwd()):
+    # dataframe to store the standarized data for each city
+    standardized_df = pd.DataFrame()
+    # read all files that end with .csv
+    if file.endswith('.csv'):
+        city = pd.read_csv(file)
+        # for loop that checks for similiar attribute names and standardizes it
```

Comment:
If the above list were to be a dict where the key is final column name and value is collection of possible source column names.
eg:
`column_map = {
"latitude"  :  {"latitude","gis_latitude", "lat"}
"longitude" : {"longitude","gis_longitude", "lng"}
}`

you can do 

`
for dst_column, src_columns in column_map.items():
  for src_column in src_columns:
    if(src_column in city.columns):
                standardized_df[dst_column] = pd.Series(city[src_column])
`
This way, you don't need new for loop for every column transformation and also if you want to extend this in future, you can just add an entry in the dict object instead of adding another for loop